src/converters/archival_object_converter.rb

```diff
@@ -68,6 +69,21 @@ def build_audit_info(object, db)
       audit_fields
     end
 
+
+    def build_dates(object, db)
+      dates = []
+
+      item = db[:item].where(:id => object[:id]).first
```

Comment:
I think we end up getting every item twice - because we already get it in the subclass. Also, there won't be an item for series' anyway. I'm probably missing something :|